content/es/docs/reference/glossary/docker.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Docker
+id: docker
+date: 2018-04-12
+full_link: https://docs.docker.com/engine/
+short_description: >
+  Docker es una tecnología de software que proporciona virtualización a nivel de sistema operativo, también conocida como contenedores.
+
+aka:
+tags:
+- fundamental
+---
+Docker (especialmente, Docker Engine) es una tecnología de software que proporciona virtualización a nivel de sistema operativo, también conocida como {{< glossary_tooltip text="contenedores" term_id="container" >}}.
+
+<!--more-->
+
+Docker utiliza características de aislamiento del kernal de Linux como cgroups y namespaces, además de sistemas de archivos con capacidad de unión como OverlayFS para permitir que contenedores independientes se ejecuten dentro de una misma instancia de Linux, evitando el coste de iniciar y mantener máquinas virtuales (VM).
```

Comment:
Is it helpful to expand “cgroups” to “cgroups (grupos de control)”?

I'm making the suggestion as this might be the reader's first encounter with that particular concept.

Reply:
@sftim *cgroups* is the name of a specific technology so in this case I don't think that makes sense to expand the concept, if readers aren't familiar with that they'll try to search for *cgroups* instead of *grupos de control* which are plenty of tutorials.

Reply:
so do I